src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -424,7 +424,21 @@ LiteralOrIri ExportQueryExecutionTrees::getLiteralOrIriFromVocabIndex(
     default:
       AD_FAIL();
   }
-};
+}
+
+// _____________________________________________________________________________
+std::optional<std::string> ExportQueryExecutionTrees::blankNodeIriToString(
+    const ad_utility::triple_component::Iri& iri) {
+  const auto& representation = iri.toStringRepresentation();
+  if (representation.starts_with(QLEVER_INTERNAL_BLANK_NODE_IRI_PREFIX)) {
+    std::string_view view = representation;
+    view.remove_prefix(QLEVER_INTERNAL_BLANK_NODE_IRI_PREFIX.size());
+    view.remove_suffix(1);
```

Comment:
I think it is cleaner to not include the leading `<` in the PREFIX, and then work with the `.getContent` method and construct it via `irirefWithoutBrackets`.
The leaking of the `StringRepresentation` should be minimized in case we want to refactor it at some point.